app/src/main/java/org/wikipedia/feed/image/FeaturedImageCardView.kt

```diff
@@ -26,9 +28,16 @@ class FeaturedImageCardView(context: Context) : DefaultFeedCardView<FeaturedImag
                 image(it.baseImage())
                 description(it.description())
                 header(it)
+                updateLabelsToWikiLang()
                 setClickListeners()
             }
         }
+
+    private fun updateLabelsToWikiLang() {
+        binding.viewFeaturedImageCardDownloadButton.text = L10nUtil.getStringForArticleLanguage("en", R.string.view_featured_image_card_download)
+        binding.viewFeaturedImageCardShareButton.text = L10nUtil.getStringForArticleLanguage("en", R.string.view_featured_image_card_share)
+    }
+
```

Comment:
Please do have these settings as a method in `FeaturedImageCard` in order to keep the consistency. Also, should we replace `en` to `wikiSite().languageCode`?